stdlib/Makefile.shared

```diff
@@ -107,6 +107,9 @@ clean::
 
 include .depend
 
+# Note that .p.cmx targets do not depend (for compilation) upon other
+# .p.cmx files.  When the compiler imports another compilation unit,
+# it looks for the .cmx file (not .p.cmx).
```

Comment:
Yes, it should be the case that the .cmx and the .p.cmx are identical.  The -p option affects assembly code generation only.